platformio/src/main.cpp

```diff
@@ -16,6 +16,32 @@ int send_command(I2C& i2c, uint8_t address, uint16_t command) {
     return i2c.write(address, cmd, sizeof(cmd));
 }
 
+Serial rn2483(p13, p14);
+
+std::string sendcmd(const char* data) {
+    printf("> %s\n", data);
+    rn2483.printf("%s\r\n", data);
+
+    std::string answer{};
+
+    // wait for max 10s
+    for (int t=0; t<100;) {
+        if (!rn2483.readable()) {
+            ++t;
+            wait(0.1);
+            continue;
+        }
+        char c = rn2483.getc();
+        if (c == '\n') {
+            break;
+        }
+        answer.push_back(c);
+    }

```

Comment:
Yes, although there is no `mut&` in C++ so the code would look like it's passed as an input parameter :wink: